mmv1/templates/terraform/custom_expand/certificate_manager_certificate_construct_full_url.go.erb

```diff
@@ -0,0 +1,38 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2017 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+func expand<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d tpgresource.TerraformResourceData, config *transport_tpg.Config) (interface{}, error) {
```

Comment:
Nit: This expander doesn't look particularly "general purpose", so I think you could hardcode most of the ruby templated pieces

Reply:
Basically everything with `<%` is a ruby template.

I am seeing the function names for all of our expanders use the same signature you have, which makes sense for handling changes to the property name (same for other uses of the property name). However, the `<%  if property.is_set -%>` I only see in one other expander, and I think it can be removed completely here. Is there a reason you expect to handle a version of the field that is a Set?

Reply:
> Is there a reason you expect to handle a version of the field that is a Set?

No, you are right. I probably thought that is's a standard check at the beginning. I will replace it with: 
```
if v == nil {
    return nil, nil
  }
``` 
as described in [guide](https://googlecloudplatform.github.io/magic-modules/develop/custom-code/#custom_expand). Let me know if this check isn't needed either.